cocos/network/HttpAsynConnection.m

```diff
@@ -73,7 +74,10 @@ - (void) connection:(NSURLConnection *)connection
     responseHeader = [[httpResponse allHeaderFields] copy];
 
     responseCode = httpResponse.statusCode;
-    if (responseCode != 200)
+    statusString = [[NSHTTPURLResponse localizedStringForStatusCode:responseCode] copy];
+    if(responseCode == 200)
+        statusString = @"OK";
+    if (responseCode/100 != 2)
```

Comment:
1.Why using magic code `200`?
2. Is `if (responseCode/100 != 2)` different from `if(responseCode != 200)`?